docs/vs-code-extension-guide/master.adoc

```diff
@@ -10,43 +10,53 @@ include::topics/templates/document-attributes.adoc[]
 
 = {VscExtensionBookName}
 
-This guide is for engineers, consultants, and others who want to use the {PluginName} for the {ProductName} ({ProductShortName}) to assist with migrating applications.
+//This guide is for engineers, consultants, and others who want to use the {PluginName} for the {ProductName} ({ProductShortName}) to assist with migrating applications.
 
 //Inclusive language statement
-include::topics/making-open-source-more-inclusive.adoc[]
+//include::topics/making-open-source-more-inclusive.adoc[]
 
-== Introduction
+// == Introduction
 
 // All these sections need to be updated
 // About {ProductName}
-include::topics/what-is-the-toolkit.adoc[leveloffset=+2]
+// include::topics/what-is-the-toolkit.adoc[leveloffset=+2]
 
 // About the VS Code plugin
-
+// include::topics/about-plugin.adoc[leveloffset=+2]
 
 // Install the Plugin
 // include::topics/installing-plugin.adoc[leveloffset=+1]
 
 // Access {ProductShortName} IDE Tools
 // include::topics/plugin-access-mta-tools.adoc[leveloffset=+1]
 //
-// // Windup IDE Plugin Components
+// Windup IDE Plugin Components
 // include::topics/plugin-components.adoc[leveloffset=+2]
 //
-// // Identify and Resolve Migration Issues with Windup
-// include::topics/plugin-identify-resolve-issues.adoc[leveloffset=+1]
-//
-// // Create a Windup Run Configuration
-// include::topics/plugin-create-config.adoc[leveloffset=+2]
+// Identify and Resolve Migration Issues with Windup
+== Analyzing your projects with the {ProductName} extension
+
+You can analyze your projects with the {ProductName} extension by creating a run configuration, running an analysis, and then reviewing and resolving migration issues detected by the analysis.
+
+//The extension interface
+include::topics/vs-code-extension-interface.adoc[leveloffset=+2]
+
+// Create a Run Configuration
+include::topics/vscode-extension-configuring-analysis.adoc[leveloffset=+2]
 //
-// // Execute Windup
-// include::topics/plugin-run-mta.adoc[leveloffset=+2]
+//  Analyzing projects
+// include::topics/vscode-extension-analyzing-projects.adoc[leveloffset=+2]
 //
-// // Review Windup Issues
-// include::topics/plugin-review-issues.adoc[leveloffset=+2]
+// Review issues
+
+== Reviewing and resolving issues with the {ProductName} extension
+
+You can review and resolve issues identified by the {ProductName} extension in the Run configuration pane.
```

Comment:
First, use the short (MTA) name for the extension. Second, "migration issues" would be more accurate.
Third, "Run configuration pane" sounds weird if the user is reviewing/resolving issues. The pane is not actually called anything. You might want to drop a specific name and just call it the right pane.